mpisppy/phbase.py

```diff
@@ -827,9 +826,12 @@ def Iter0(self):
                 stochastic program with the nonanticipativity constraints
                 removed.
         """
+
+        # JPW: the pre iter0 callback should arguably be called not from within
```

Comment:
There are multiple pathways to iter0, so I would argue that this is the best place. But, in any event, this comment should probably go somewhere else (e.g. an issue)

Reply:
Fair - corrected (and opened an issue).